core/src/main/scala/janstenpickle/scala/syntax/syntax.scala

```diff
@@ -13,8 +12,10 @@ import uscala.result.Result.{Fail, Ok}
 
 import scala.concurrent.{ExecutionContext, Future}
 
-object catsconversion {
-  implicit def toResult[A, B](xor: Xor[A, B]): Result[A, B] = xor.fold(Result.fail, Result.ok)
+object CatsConversion {
+  implicit def toResult[A, B](xor: Either[A, B]): Result[A, B] = xor.fold(
+    Result.fail, Result.ok
+  )
 
 
   import cats.Applicative
```

Comment:
Oops, I managed to commit some play code for doing something else here, I'll remove it.